flask_swagger.py

```diff
@@ -171,7 +171,7 @@ def spec():
         endpoint = app.view_functions[rule.endpoint]
         methods = dict()
         for verb in rule.methods.difference(ignore_verbs):
-            if hasattr(endpoint, 'methods') and verb in endpoint.methods:
+            if hasattr(endpoint, 'methods') and verb in endpoint.methods and hasattr(endpoint.view_class, verb.lower()):
                 verb = verb.lower()
                 methods[verb] = getattr(endpoint.view_class, verb)
```

Comment:
Good catch!

I'm a little anal about repetitive code so can I suggest the following:

```
if hasattr(endpoint, 'methods'):
    verb = verb.lower()
    method = getattr(endpoint.view_class, verb, None)
    if method is not None:
        methods[verb] = method
```